ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -1570,3 +1568,25 @@ func sumGPUs(resources map[corev1.ResourceName]resource.Quantity) resource.Quant
 
 	return totalGPUs
 }
+
+type clusterConfig struct{}
```

Comment:
```suggestion
type clusterConfigCtxKey struct{}
```

Reply:
How about clarifying the purpose of this struct by adding something like CtxKey?

Reply:
Yes it should be renamed. Kind of missed it when refactoring.